fog/view/server/src/bin/main.rs

```diff
@@ -63,11 +69,9 @@ fn main() {
     );
     server.start();
 
-    let config2 = config.clone();
-    let get_config_json = Arc::new(move || {
-        serde_json::to_string(&config2)
-            .map_err(|err| RpcStatus::with_message(RpcStatusCode::INTERNAL, format!("{:?}", err)))
-    });
+    let config_json = serde_json::to_string(&config)
```

Comment:
Consider panicing, this should never happen and if it does it indicates something questionable with the configuration.